library/addisco.c

```diff
@@ -155,7 +155,7 @@ get_string (unsigned char *beg,
             unsigned char *end,
             unsigned char **at)
 {
-	char buffer[HOST_NAME_MAX];
+	char buffer[75];
```

Comment:
why 75 ?

Reply:
chose it arbitrarily, gives us enough space for our max room length

Reply:
Might be worth a comment to explain this for the person who is supporting this in late 2021 and beyond.

Reply:
sure thing